operator/pkg/awsprovider/launchtemplate/reconciler.go

```diff
@@ -96,35 +101,32 @@ func (c *Controller) createLaunchTemplate(ctx context.Context, dataplane *v1alph
 		return fmt.Errorf("getting control plane ca certificate, %w", err)
 	}
 	_, clusterCA := secrets.Parse(caSecret)
-	paramOutput, err := c.ssm.GetParameterWithContext(ctx, &ssm.GetParameterInput{
-		Name: aws.String("/aws/service/eks/optimized-ami/1.20/amazon-linux-2/recommended/image_id"),
-	})
+	amiID, err := c.amiID(ctx, dataplane)
 	if err != nil {
-		return fmt.Errorf("getting ssm parameter, %w", err)
+		return fmt.Errorf("getting ami id for worker nodes, %w", err)
 	}
-	amiID := *paramOutput.Parameter.Value
 	input := &ec2.CreateLaunchTemplateInput{
 		LaunchTemplateData: &ec2.RequestLaunchTemplateData{
```

Comment:
Is this way of creating nodes using launch templates approved by finance team internally for our extensive testing purposes ?

Reply:
Will sync with you offline on this.